src/pytorch_lightning/_graveyard/cloud_io.py

```diff
@@ -0,0 +1,23 @@
+import pickle
+import warnings
+from copy import deepcopy
+from typing import Any
+
+
+class RedirectingUnpickler(pickle.Unpickler):
+    def find_class(self, module: str, name: str) -> Any:
+        if module.startswith("pytorch_" + "lightning"):
+            # for the standalone package this won't do anything,
+            # for the unified mirror package it will redirect the imports
+
+            old_module = deepcopy(module)
```

Comment:
Why are we deepcopying if it's a plain string?

Reply:
This is a leftover, where `module` was changed in place. Should be safe to delete now.